src/airobot/robot/ur5e_pybullet.py

```diff
@@ -0,0 +1,721 @@
+"""
+Pybullet simulation environment of a UR5e
+ robot with a robotiq 2f140 gripper
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import copy
+import threading
+import time
+
+import numpy as np
+import pybullet as p
+import pybullet_data
+from gym.utils import seeding
+
+from airobot import arutil
+from airobot.robot.robot import Robot
+
+
+class UR5eRobotPybullet(Robot):
+    """
+    Class for the pybullet simulation environment
+        of a UR5e robot with a robotiq 2f140 gripper.
+    """
+
+    def __init__(self, cfgs, render=False, seed=None, self_collision=False):
+        """
+        Constructor for the pybullet simulation environment
+        of a UR5e robot with a robotiq 2f140 gripper
+
+        Args:
```

Comment:
@taochenshh what style guide are you using?

Reply:
Google Python style.

Reply:
The code passes Flask8